### ISSUE TITLE
Getting times from timeline

### ISSUE THREAD

**Issue by kay-is** [owickstrom/komposition#64]
There seems to be no way to find out when a clip starts/ends.

Example:

I add three audio-clips with two gaps.

Now I want to add the video clips so they start at the right timings of the audio clips.

At the moment I have to fiddle around with video-gaps until they have the right length and then add the video-clip.

**Comment by owickstrom** [comment]
Yeah, you're absolutely right. I've been thinking about adding some kind of sidebar with information about the currently focused part of the timeline. This could show durations and timespans of each part, and when you're focusing a single clip the information about the source media. Would that help?

Also, I've been considering some sort of time ruler displayed along the timeline. It's more crude, but might give a hint about the duration of parts in the timeline.
